content/gocd_on_kubernetes/setup_and_configuration.md

```diff
@@ -59,12 +59,19 @@ Helm has two parts to it, the `helm` client and a server called `Tiller`.
 
 The helm client is a CLI that let's you install and update packaged applications on Kubernetes. Helm's [installtion documentation](https://github.com/helm/helm#user-content-install) details various ways to install the Helm client.
 
+### Start the Kubernetes cluster
+if running locally -
+minikube start
```

Comment:
@infomaven - https://docs.gocd.org/current/gocd_on_kubernetes/setup_and_configuration.html : step 1 talking about the setup (includes minikube start and links to guides for popular K8s service providers like GKE, EKS, etc.) I don't see why minikube start should be added in again later.

Reply:
A reminder doesn't hurt...especially if someone if not very familiar with Kubernetes. But in any case, I just noticed a typo in line 60 directly above this change.   It should be *lets* ...not *lets* with an apostrophe.